src/movementHandler.ts

```diff
@@ -0,0 +1,123 @@
+import { Collidable } from './collidable';
+import { FacingAngles } from './facingAngles';
+import { PhysObject } from './physObject';
+
+export class MovementHandler {
```

Comment:
Like my other comments, I'd recommend exporting the functions directly. Having a class without any state to encapsulate doesn't seem worthwhile.

Reply:
done